components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -331,14 +331,16 @@ def render_thumbnail(request, iid, w=None, h=None, conn=None, _defcb=None,
     """
     server_id = request.session['connector'].server_id
     direct = True
+    defaultSize = conn.getDefaultThumbnailSize()
```

Comment:
It will just be requested from the server the first time the method is called, the value is held in `_defaultThumbnailSize` variable in the Gateway